libsolidity/InterfaceHandler.cpp

```diff
@@ -16,7 +16,7 @@ InterfaceHandler::InterfaceHandler()
 	m_lastTag = DocTagType::None;
 }
 
-unique_ptr<string> InterfaceHandler::getDocumentation(
+string InterfaceHandler::getDocumentation(
```

Comment:
Return a const reference to prevent copy?
In the error case at the end, you can return e.g. the user documentation.